src/views/v2/month.php

```diff
@@ -18,9 +18,34 @@
 $this->template( 'events-bar' );
 
 $this->template( 'top-bar' );
-
 ?>
 
-<div class="tribe-events-calendar-month">
-	<h1>Welcome to the month view</h1>
-</div>
\ No newline at end of file
+<h2 class="tribe-common-a11y-hidden" id="tribe-calendar-header"><?php printf( esc_html__( 'Calendar of %s', 'the-events-calendar' ), tribe_get_event_label_plural() ); ?></h2>
+
+<div class="tribe-events-calendar-month" role="grid" aria-labelledby="tribe-calendar-header" aria-readonly="true">
+
+	<?php $this->template( 'month/grid-header' ); ?>
+
+	<div class="tribe-events-calendar-month__body" role="rowgroup">
+
+		<?php // @todo: replace this with the actual month days. Using these for(s) for presentation purposes. ?>
+		<?php for ( $week = 0; $week < 4; $week++ ) : ?>
+
+			<div class="tribe-events-calendar-month__week" role="row">
+
+				<?php for ( $day = 0; $day < 7; $day++ ) : ?>
+
+					<div class="tribe-events-calendar-month__day" role="gridcell" tabindex="-1">
```

Comment:
I know we are following the pattern that gcal has established, which is great, however I'm curious if we have any grand designs in mind as of now that would result in needing to implement these `tabindex="-1"` attributes? If not, we can likely nix them for now and add them later and as we would need them for our implementation ; )

We should follow along with gcal and relate the gridcell to the day heading, so `aria-labelledby` on this and then a matching/corresponding `id` on the day heading. This will add additional context to this wrapper.

One other thought, I would recommend including this day wrapper as part of the day template partial itself, as that will nicely organize that as a completely self-contained unit, especially in the case where we may need to do various conditional logic for days. Just my two cents!